app/gui/src/ide/initializer.rs

```diff
@@ -104,6 +105,32 @@ impl Initializer {
         std::mem::forget(executor);
     }
 
+    fn register_views(app: &Application) {
+        app.views.register::<ide_view::root::View>();
+        app.views.register::<ide_view::graph_editor::GraphEditor>();
+        app.views.register::<ide_view::graph_editor::component::breadcrumbs::ProjectName>();
+        app.views.register::<ide_view::code_editor::View>();
+        app.views.register::<ide_view::project::View>();
+        app.views.register::<ide_view::searcher::View>();
+        app.views.register::<ide_view::welcome_screen::View>();
+        app.views.register::<ensogl_component::text::Area>();
+        // FIXME: decimal_aligned is a private module:
+        // app.views.register::<ensogl_component::selector::decimal_aligned::FloatLabel>();
+        app.views.register::<ensogl_component::selector::NumberPicker>();
+        app.views.register::<ensogl_component::selector::NumberRangePicker>();
+
+        // TODO(akavel): add warning in ListView.label()'s docs
+        // As long as .label() of a View is the same, shortcuts and commands are currently also
+        // expected to be the same, so it should not be important which concrete type parameter of
+        // ListView we use below.
+        app.views.register::<ensogl_component::list_view::ListView<ensogl_component::list_view::entry::Label>>();
```

Comment:
Can we use `ListView<()>` here instead of concrete type?

Reply:
Ah, I was too fast and forgot to save & compile: `()` does not implement `Entry` trait :(

Reply:
Okay, we can leave it as it is then :)